src/frontends/pytorch/src/op/pixel_shuffle.cpp

```diff
@@ -67,6 +69,38 @@ OutputVector translate_pixel_shuffle(const NodeContext& context) {
     return {context.mark_node(std::make_shared<v1::Reshape>(transpose, shape_after, false))};
 };
 
+OutputVector translate_channel_shuffle(const NodeContext& context) {
+    // aten::channel_shuffle(Tensor self, int groups) -> Tensor
+    num_inputs_check(context, 2, 2);
+    auto x = context.get_input(0);
+    auto groups = context.get_input(1);
+    auto neg_1 = context.mark_node(v0::Constant::create(element::i32, Shape{1}, {-1}));
+    auto zero = context.mark_node(v0::Constant::create(element::i32, Shape{}, {0}));
+    auto one = context.mark_node(v0::Constant::create(element::i32, Shape{}, {1}));
+    Output<Node> shape;
+    Output<Node> rank;
+    std::tie(shape, rank) = get_shape_rank(context, x, true);
+    // PyTorch realization uses assumption that channels dim is always 1
+    auto indices = context.mark_node(v0::Constant::create(element::i32, Shape{2}, {0, 1}));
+    auto dims = context.mark_node(std::make_shared<v8::Gather>(shape, indices, zero));
+    auto dims_splitted = context.mark_node(std::make_shared<v1::Split>(dims, zero, 2));
+    auto c = dims_splitted->output(1);
+    auto n = dims_splitted->output(0);
+    auto rank_1d = context.mark_node(std::make_shared<v0::Unsqueeze>(rank, zero));
```

Comment:
Please use `get_shape_rank` helper with `false` to have 1d rank value

Reply:
current realization does not use rank at all, so removed this line